### ISSUE TITLE
[Feature request] Free zoom with pinch gesture

### ISSUE THREAD

**Issue by ov1d1u** [abdallahmehiz/mpvKt#33]
Add a pinch gesture to allow zooming the video. Ideally like in a image viewer, which allows to zoom into certain parts of the image.

**Comment by toutoumu** [comment]
This function is great. It is best to set double-click to zoom in and double-click to zoom out.

**Comment by johnnyw94** [comment]
Yeah this will be a great feature. I love this one in MX Player. Kindly add this to mpvKt, I'll use it as a daily driver.

**Comment by rg991** [comment]
Yes, I also want this

**Comment by GhostGH** [comment]
I think YouTube's pinch-to-zoom is a great example

**Comment by Aryan-Chauhan** [comment]
I also want this.

I think Just Player supports this.

**Comment by abdallahmehiz** [comment]
Attempted to add this but `detectTransformGestues`, the function that that detects zoom and pinch from Jetpack Compose, consumes all gestures if active.
I'd have to find a better way to detect and handle gestures.

**Comment by johnnyw94** [comment]
> Attempted to add this but `detectTransformGestues`, the function that that detects zoom and pinch from Jetpack Compose, consumes all gestures if active.
> I'd have to find a better way to detect and handle gestures.

Thank you, Fingers crossed!

**Comment by GhostGH** [comment]
> Attempted to add this but `detectTransformGestues`, the function that that detects zoom and pinch from Jetpack Compose, consumes all gestures if active. I'd have to find a better way to detect and handle gestures.

I just noticed that open-source android gallery "Aves Libre" has free zoom with pinch gesture. Maybe you could check out their code to find their implementation of this feature?
